src/database/migrations/2019_10_16_100000_create_calendars_table.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateCalendarsTable extends Migration
+{
+    public function up()
+    {
+        Schema::create('calendars', function (Blueprint $table) {
+            $table->increments('id');
+
+            $table->string('name');
+            $table->string('color');
+            $table->boolean('private');
+
+            $table->integer('created_by')->unsigned()->index()->nullable();
+            $table->foreign('created_by')->references('id')->on('users');
+
+            $table->timestamps();
+        });
+
+        Schema::table('events', function (Blueprint $table) {
```

Comment:
the calendars migration should run before the calendar_events, so that we wouldn't need to make these updates due to the migrations' order